link-verifier/verify-links.py

```diff
@@ -166,7 +170,7 @@ def parse_file(html_file):
     return HtmlFile(html_file)
 
 def html_name_from_markdown(filename):
-    md_pattern = re.compile('\.md', re.IGNORECASE)
+    md_pattern = re.compile("\.md$", re.IGNORECASE)
```

Comment:
Not sure why this uses this, compared to the global one at the top of the file, but just tried using quotes to see if that helped with the warning

Added the $ to make sure it only looks for files that fully end in `.md`